test/system/src/main/java/io/pravega/test/system/framework/Utils.java

```diff
@@ -24,6 +41,34 @@ public static String getConfig(final String key, final String defaultValue) {
         return System.getenv().getOrDefault(key, System.getProperty(key, defaultValue));
     }
 
+    public static Service createServiceInstance(final String serviceType, final URI zkUri, final URI hdfsUri, final  URI contUri) {
```

Comment:
We could break this separate methods likes createZookeeperService(), createBookkeeperService() and so on.
This would further simplify the test code...

Reply:
done.